lib/mix/triplex.ex

```diff
@@ -29,7 +29,7 @@ defmodule Mix.Triplex do
   def migrations_path(repo) do
     repo
     |> source_repo_priv()
-    |> Path.join("tenant_migrations")
+    |> Path.join(config().migrations_path || "tenant_migrations")
```

Comment:
i think will be better set the default value in the config struct, so we don't need repeat this line in Triplex module too.
```suggestion
    |> Path.join(config().migrations_path)
```